metadata.md

```diff
@@ -0,0 +1,48 @@
+# Metadata
+At the top of every feature detect in Modernizr there is a JSON fragment that represents the metadata of the test. This data is used to build the webpage, for example.
+This is an example of this schema and does not represent a real test: 
+## Schema
+```json
+/*!
+{
+  "name": "JPEG 2000",
+  "property": "jpeg2000",
+  "tags": ["media", "attribute"],
+  "caniuse": "jpeg2000",
+  "authors": ["Markel Ferro (@MarkelFe)", "@rejas", "Brandom Aaron"],
+  "builderAliases": ["a_download"],
+  "polyfills": ["xaudiojs"],
+  "aliases": ["jpeg-2000", "jpg2"],
+  "async": true,
+  "warnings": ["These tests currently require document.body to be present"],
+  "knownBugs": ["This will false positive in IE6"],
+  "notes": [{
+    "name": "Specification",
+    "href": "https://www.w3.org/"
+  }, {
+    "name": "Github issue",
+    "href": "https://github.com/Modernizr/"
+  }]
+}
+!*/
+/* DOC
+Here it would go a description of the feature test. You can use **markdown** here :)
+*/
+```
+> There is no set order in which they must appear, but it is common to see `name` and `property` at the top while `notes` at the bottom.
+
+## Item description
+|                  | Necesity |                      Description                     |                           Notes                           |
+|------------------|:--------:|:----------------------------------------------------:|:---------------------------------------------------------:|
+| `name`           | required |             Name of the feature detection            |                                                           |
+| `property`       | required | The property name established in `Modernizr.addTest` |     Preferably just lowercase, without any punctuation    |
+| `tags`           | optional |    A group that encapsulates many feature detects    |                                                           |
+| `caniuse`        | optional |      A conversion table of caniuse and Modernizr     | Consider adding it to test/browser/integration/caniuse.js |
+| `authors`        | optional |          List of contributors of the script          |          There are a couple of ways to express it         |
+| `builderAliases` | optional |                                                      |                                                           |
+| `polyfills`      | optional |     Available polyfills for not working versions     |               Add them in lib/polyfills.json              |
+| `aliases`        | optional |         Other names that the feature goes by         |                                                           |
```

Comment:
I'm not completely sure what are aliases used for, I think that their main function is to show them easier in the search, but maybe they have some kind of functionality that you may call the function by either of the 2 names. Just wanted to make sure.

Reply:
they should not be used for anything new. its a last resort for if something has multiple terms or things that the author thought people would use instead of the canonical name

Reply:
Maybe adding a _recommended_ or _not recommended_ option in the necessity column would be nice?